app/dashboard/become-provider/application/provider-profile-form.tsx

```
@@ -0,0 +1,194 @@
+          loading: 'Creating your provider profile...',
+          success: () => {
+            // Navigate to the next step after the toast is shown
+            setTimeout(() => {
+              router.push(`/dashboard/become-provider/application/images?providerId=${response.id}`);
+            }, 500);
+            return 'Provider profile created successfully!';
+          },
+          error: 'Failed to create provider profile',
+        }
+      );
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)